src/com/facebook/buck/android/NonPreDexedDexBuildable.java

```diff
@@ -178,7 +178,7 @@
       boolean desugarInterfaceMethods) {
     super(buildTarget, filesystem);
     this.androidPlatformTarget = androidPlatformTarget;
-    this.additionalJarsForProguard = additionalJarsForProguard;
```

Comment:
Does `this.additionalJarsForProguard` ever get assigned or we don't use it at all?

Reply:
Its been renamed to make the purpose clear. We use it below